### PR TITLE
Bumped version to 3.0.

### PR DESCRIPTION
**Pull request by AWSHurneyt**
Signed-off-by: AWSHurneyt <hurneyt@amazon.com>

### Description
Bumped version to 3.0.
 
### Issues Resolved
https://github.com/opensearch-project/alerting-dashboards-plugin/issues/290
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AWSHurneyt** [comment]
The cypress failures relating to `ClusterMetricsMonitors` were failing due to a bug introduced in common-utils 3.0. This PR fixes that bug https://github.com/opensearch-project/common-utils/pull/359. Those cypress tests should succeed once the new 3.0 common-utils artifact is available.

**Comment by AWSHurneyt** [comment]
The test workflows are failing with the following error:
```
@opensearch-project/opensearch@2.1.0: The engine "yarn" is incompatible with this module. Expected version "^1.22.10". Got "1.21.1"
```
The changes in this PR don't seem to be the cause of that error though according to this issue. A solution is still being discussed.
https://github.com/opensearch-project/opensearch-build/issues/3210

**Comment by lezzago** [comment]
The problem issue has been merged, so will need to update this again.

**Comment by AWSHurneyt** [comment]
The cypress tests workflow appears to be failing because the first few test suites are being executed before the domain has come up. However, because we don't guarantee stability on the `main` branches, we can review and merge in this bump, and resolve the workflow failures in a separate PR.

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -7,9 +7,9 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.x'
-  OPENSEARCH_VERSION: '2.7.0-SNAPSHOT'
-  ALERTING_PLUGIN_BRANCH: '2.x'
+  OPENSEARCH_DASHBOARDS_VERSION: 'main'
+  OPENSEARCH_VERSION: '3.0.0-SNAPSHOT'
```
Comment (bowenlan-amzn) [inline .github/workflows/cypress-workflow.yml:11]: Recommend to remove the usage of this variable

**Review by bowenlan-amzn — APPROVED** [review]

**Review by getsaurabh02 — APPROVED** [review]

File: .github/workflows/cypress-workflow.yml (hunk 1 above)
Comment (AWSHurneyt) [reply .github/workflows/cypress-workflow.yml:11]: Ah, thanks for the tip! I'll add this as a follow-up action.

**Pull request merged**